app/models/like.rb

```diff
@@ -1,4 +1,8 @@
 class Like < ApplicationRecord
-  belongs_to :user
-  belongs_to :post
+  belongs_to :author, class_name: 'User'
+  belongs_to :post, class_name: 'Post'
+
+  def update_likes_counter
+    post.increment!(Likes_Counter)
+  end
```

Comment:
### OPTIONAL
*You can use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) -* `after_save` to call the `#update_likes_counter` method every time a new post has been saved to database, then make the `update_likes_counter` method private

```ruby
after_save :update_likes_counter

  private

  def update_likes_counter
    post.increment!(:likes_counter)
  end
```